src/generators/css-filter.ts

```diff
@@ -19,10 +19,10 @@ export enum FilterMode {
  * Bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=501582
  * Patch: https://chromium-review.googlesource.com/c/chromium/src/+/1979258
  */
-export function hasChromiumIssue501582() {
+export function hasPatchForChromiumIssue501582() {
     return Boolean(
         isChromium &&
-        compareChromeVersions(chromiumVersion, '81.0.4035.0') >= 0
+        compareChromeVersions(chromiumVersion, '81.0.4035.0') <= 0
```

Comment:
`compareChromeVersions` will return `0` if it's the same. Will return `1` if(this case) `chromiumVersion` is bigger and `-1` if `chromiumVersion` is less.

So you'd want to still invert this comparison to  `>=` as if chromiumVersion is bigger or the same(so `-1` or `0` as return values) return true.

Reply:
Thank you for catching that!

Reply:
I think this might have been happening because the "dark" background was being set as:
https://github.com/darkreader/darkreader/blob/fb9b87a0efc8e0c6e7d3ad4a131859561b9b8439/src/generators/css-filter.ts#L83
whereas the inverse or `rgb(0, 0, 0)` really isn't very dark.

I went ahead and changed that, but I think I see now that you intended for `hasPatchForChromiumIssue501582()` to mean that Chromium has patched the issue and not that we should. In that case, I'll go ahead and invert the comparison as well as the logic below:
https://github.com/darkreader/darkreader/blob/fb9b87a0efc8e0c6e7d3ad4a131859561b9b8439/src/generators/css-filter.ts#L81-L84